javaparser-core-testing/src/test/java/com/github/javaparser/PositionTest.java

```diff
@@ -20,4 +21,21 @@ public void testOrIfInvalid() {
         assertEquals(invalid2, invalid2.orIfInvalid(invalid));
     }
 
+    @Test
+    public void testPositionExceptionMessage() {
```

Comment:
Can you use assertThrows instead of try/catch?
Is the intent of the test to verify the error message or to ensure that an incorrect instance of the Position class cannot be created.
The method name should describe the intent of the test.

Reply:
Yes sure, this change has been made. The intention of this test is to verify the exception message format to alleviate concerns from this comment  https://github.com/javaparser/javaparser/blob/master/javaparser-core/src/main/java/com/github/javaparser/Position.java#L64